contract_comm/blockchain_parameters/blockchain_parameters.go

```diff
@@ -67,21 +67,36 @@ const (
 		"payable": false,
 		"stateMutability": "view",
 		"type": "function"
-	  },
-	  {
+	},
+	{
 		"constant": true,
 		"inputs": [],
-		"name": "intrinsicGasForAlternativeFeeCurrency",
+		"name": "getUptimeLookbackWindow",
 		"outputs": [
-		  {
-			"name": "",
-			"type": "uint256"
-		  }
+			{
+				"internalType": "uint256",
+				"name": "lookbackWindow",
+				"type": "uint256"
+			}
 		],
 		"payable": false,
 		"stateMutability": "view",
 		"type": "function"
-	  }
+	},
+	{
+	"constant": true,
+	"inputs": [],
+	"name": "intrinsicGasForAlternativeFeeCurrency",
+	"outputs": [
+		{
+		"name": "",
+		"type": "uint256"
+		}
+	],
+	"payable": false,
+	"stateMutability": "view",
+	"type": "function"
+	}
```

Comment:
The whitespace here is messed up (lines 87-98 should be indented relative to the braces, and 92-93 indented relative to their braces).   Though it was already messed up to some degree before this PR (consistenting of a mix of 2-space indents and tab indents).